laws/src/main/scala/cats/laws/InvariandChoosableLaws.scala

```diff
@@ -0,0 +1,30 @@
+package cats
+package laws
+
+import cats.data.INothing
+
+/**
+ * Laws that must be obeyed by any `cats.InvariantChoosable`.
+ */
+trait InvariantChoosableLaws[F[_]] extends InvariantChoiceLaws[F] {
```

Comment:
I notice only now that there is a typo in the file name. It should probably be `InvariantChoosableLaws`.

Reply:
Great catch, thanks!